CRM/MembershipExtras/Hook/Pre/MembershipEdit.php

```diff
@@ -157,6 +158,50 @@ public function preventExtendingPaymentPlanMembership() {
     }
   }
 
+  /**
+   * Prevents editing the active Membership if its contribution has another
+   * payment.
+   *
+   * Usually recording the payment for the membership's contribution means :
```

Comment:
This is the case only for non payment-plans payments  , it worth noting that here.